cni/network/network.go

```diff
@@ -1011,25 +955,21 @@ func (plugin *NetPlugin) Delete(args *cniSkel.CmdArgs) error {
 		k8sNamespace string
 		networkID    string
 		nwInfo       network.EndpointInfo
-		cniMetric    telemetry.AIMetric
 	)
 
-	startTime := time.Now()
-
 	logger.Info("Processing DEL command",
 		zap.String("containerId", args.ContainerID),
 		zap.String("netNS", args.Netns),
 		zap.String("ifName", args.IfName),
 		zap.Any("args", args.Args),
 		zap.String("path", args.Path),
 		zap.ByteString("stdinData", args.StdinData))
-	sendEvent(plugin, fmt.Sprintf("[cni-net] Processing DEL command with args {ContainerID:%v Netns:%v IfName:%v Args:%v Path:%v, StdinData:%s}.",
-		args.ContainerID, args.Netns, args.IfName, args.Args, args.Path, args.StdinData))
 
 	defer func() {
 		logger.Info("DEL command completed",
 			zap.String("pod", k8sPodName),
 			zap.Error(log.NewErrorWithoutStackTrace(err)))
+		telemetry.SendEvent(fmt.Sprintf("DEL command completed: [released ip]: %+v [podname]: %s [namespace]: %s [error]: %v", nwCfg.IPAM.Address, k8sPodName, k8sNamespace, err))
```

Comment:
same as above. if `logger` already writes to the file, then we are just duplicating data, as well as going away from structured logs.

Reply:
see above-- I was trying to replicate what I was seeing in the cnslogger but since we log before anyway this isn't helpful